TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -83,6 +83,22 @@ const BenchmarkMetricRegex = {
             },
         }
     },
+    Score: {
+        metrics: {
+            "Score":{
+                //Example: ArrayListSortComparatorBench score: 2757481.000000 (2.757M 1483.0%)
+                regex: /ArrayListSort[\s\S\n]*? score: ?(\d*\.?\d*)/,
+                higherbetter: true,
+                units: "kb",
```

Comment:
Wrong units. It should be `iterations/batch`. You can refer to https://github.com/AdoptOpenJDK/bumblebench/wiki/BumbleBench-and-HumbleBench#quick-start.